README.md

```diff
@@ -43,176 +29,154 @@ Where table headings (`th`) are used as row headings, `scope="row"` attributes m
-
-```sass
-.my-table {
-	@include oTableBase;
-}
-```
-
-#### Variant classes and placeholders
-
-Additional classes may be added to the table root element to also apply the following styling options. They can be combined as required.
-
-#### Content styles
-
-Class: `o-table__cell--content-secondary`, Mixin: `oTableCellContentSecondary`
-
-Reduce the size of some text in a cell and display block to start a new line. The class should be applied to a `<span>` or `<div>` element inside of the table cell.
+#### Table Utility Styles
+- `o-table--row-stripes` - Apply to the table for alternating stripes on the table rows.
+- `o-table__cell--numeric` - Apply to numeric cells to align content to the right.
+- `o-table__cell--vertically-center` - Apply to cells which should center vertically.
 
-#### Cell alignment
+See more styles in the registry: [o-table demos](https://registry.origami.ft.com/components/o-table).
 
-Class: `o-table__cell--numeric`, Mixin: `oTableCellNumeric`
 
-Right-align the cell contents.
-
-Class: `o-table__cell--vertically-center`, Mixin: `oTableCellVerticallyCenter`
-
-Vertically center the cell contents.
-
-#### Row stripes
-
-Class: `o-table--row-stripes`, Mixin: `oTableRowStripes`
-
-A background colour will be set on the whole table, and alternate rows within the `tbody` will have their background colour set to a pink tint.
-
-#### Horizontal lines
-
-Class: `o-table--horizontal-lines`, Mixin: `oTableHorizontalLines`
-
-Thin lines will be rendered under each `td` element giving the appearance of lines between rows.
-
-#### Vertical lines
-
-Class: `o-table--vertical-lines`, Mixin : `oTableVerticalLines`
-
-Thin lines will be rendered to the left and right of each `td` element giving the appearance of lines between columns.
-
-#### Responsive
-
-There are three responsive options available for displaying data in a table.
-
-##### Flat
-
-Class: `o-table--responsive-flat`, Mixin: `oTableResponsiveFlat`
-
-Using the Responsive Flat version will render the table to change at narrow viewpoints into a row-based table with each row having a duplicate of the table headers on the left side. This uses JavaScript to inject the headers into each row.
-
-Please note that this option will not work in *Core* experience.
-
-##### Scroll
-
-Class: `o-table--responsive-scroll`, Mixin: `oTableResponsiveScroll`
-
-On a narrow viewpoint the Responsive Scroll version will move the headers to the right hand side of the table, and be fixed. This allows the data to be displayed in a column format which would allow the user to swipe left or right going through the data.
+#### Silent mode
 
-##### Overflow
+If using `o-table` in silent mode, `@include oTableAll` includes every feature. Alternatively features may be includes granularly:
 
-Class: `o-table--responsive-overflow`, Mixin: `oTableResponsiveOverflow`
+Required:
+```scss
+// Basic styles and utilities.
+@include oTableBase;
```

Comment:
I mas have misread your explanation in the readme about the granularity, but I'm not sure I understand 100%

Reply:
The ability to use custom classes has been removed (reasoning in the PR description). I was trying to express that to include `o-table` features granularly.

Do this:
```scss
@include oTableBase;
@include oTableWrapper;
@include oTableContainer;
@include oTableResponsiveFlat;
...
```

Instead of this:
```scss
.o-table {
    @include oTableBase;
    @include oTableResponsiveFlat;
}
.o-table-wrapper {
    @include oTableWrapper;
}
.o-table-container {
    @include oTableContainer;
}
...
```

I only spoke to Jake and a few others casually about this, let me know if you have any questions or objections.